File: 1014 - consumo/consumo.kt
import java.util.*

fun consumo( x: Int, y : Double) : Double{
    return ((x.toDouble())/ y);
}

fun main( args: Array<String>){
    var input = Scanner(System.`in`)
    var x : Int
    var y : Double
    x = input.nextInt();
    //y = readLine()!!.toDouble()  - works
    y = input.nextDouble() //not works
    println("%.3f km/l". format(consumo(x,y)))
}

File: 1014 - consumo/consumo.py
""" python 2 """

def consumo(x, y):
    return(x / y)

x = int(input())
y = float(input())
print '{:.3f} km/l'.format( consumo(x , y) )

""" python 3 """


def consumo(x, y):
    return(x / y)

x = int(input())
y = float(input())
print ('{:.3f} km/l'.format( consumo(x , y) ))